cookiecutter.json

```diff
@@ -4,7 +4,7 @@
     "__directory_name": "{{ cookiecutter.project_name.lower().replace(' ', '_') }}",
     "author_name": "Your Name",
     "dependency_manager": ["pip", "poetry"],
-    "data_version_control": ["yes", "no"],
```

Comment:
Good idea! Could you change this to `use_data_version_control` for those who are not aware of DVC?